azure-webapp-maven-plugin/README.md

```diff
@@ -1,186 +1,179 @@
 # Maven Plugin for Azure App Service
+
+## Table of Content
+  - [Overview](#overview)
+  - [Prerequisites](#prerequisites)
+  - [Quick Start](#quick-start)
+  - [Goals](#goals)
+      - [`azure-webapp:config`](#azure-webappconfig)
+      - [`azure-webapp:deploy`](#azure-webappdeploy)
+  - [Configuration Details](#configuration-details)
+      - [`<javaVersion>`](#javaversion)
+      - [`<webContainer>`](#webcontainer)
+      - [`<resource>`](#resource)
+      - [`<region>`](#region)
+      - [`<pricingTier>`](#pricingtier)
+  - [Samples](#samples)
+  - [Feedback and Questions](#feedback-and-questions)
+  - [Data and Telemetry](#data-and-telemetry)
+
+## Overview
 [![Maven Central](https://img.shields.io/maven-central/v/com.microsoft.azure/azure-webapp-maven-plugin.svg)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.microsoft.azure%22%20AND%20a%3A%22azure-webapp-maven-plugin%22)
 
-The Maven Plugin for Azure App Service provides seamless integration into Maven projects, 
-and makes it easier for developers to deploy to different kinds of Azure Web Apps:
+The Maven Plugin for Azure App Service provides seamless integration into Maven projects,
+and makes it easier for developers to deploy Web Apps to [Azure App Service](https://docs.microsoft.com/en-us/azure/app-service/):
```

Comment:
Suggest changing to "... and makes it easier to deploy your applications to following types of Azure App Service."

1) The term of "Web Apps" could be confusing. 
2) The type is specified as follows.

Reply:
Done, please have a check and see if there is anything else need to be improved